proto/proto-backplane-grpc/src/main/proto/deephaven/proto/application.proto

```diff
@@ -45,25 +45,12 @@ message FieldsChangeUpdate {
  * A lightweight object describing the exposed field.
  */
 message FieldInfo {
-  Ticket ticket = 1;
+  TypedTicket ticket = 1;
   string field_name = 2;
-  FieldType field_type = 3;
+//  string field_type = 3;
```

Comment:
remove this, renumber fields, since we're already breaking things?

and while we're at it would you mind capitalizing the listFields method to ListFields?